### PR TITLE
feat(mok-example/FrontEndRecipe): add mock example for api index

### PR DESCRIPTION
**Pull request by josefinalliende**
### Context
In a previous PR we excluded `.mock` files from the `tsconfig.json` . Now we want to add a very simple example of a mock

### What has been done
A mock example file for the `api/index.ts` was added

### In particular you have to check
- The example mok `index.mock.ts`
- The `frontend recipe` where this file was copied

### CODE REVIEW

File: spec/features/front_end_spec.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require "spec_helper"
 
-RSpec.describe "Front end" do
+RSpec.fdescribe "Front end" do
```
Comment (monkeyci) [inline spec/features/front_end_spec.rb:3]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] RSpec/Focus: Focused spec found.

Comment (monkeyci) [inline spec/features/front_end_spec.rb:3]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/MultipleMemoizedHelpers: Example group has too many memoized helpers [9/5]

Comment (josefinalliende) [reply spec/features/front_end_spec.rb]: upps

Comment (josefinalliende) [reply spec/features/front_end_spec.rb]: In other project we disabled this rule... what should I do here?

File: spec/features/front_end_spec.rb [hunk 2]
```diff
@@ -6,14 +6,14 @@
     remove_project_directory
   end
 
-  let(:application_css_path) { "#{project_path}/app/javascript/css/application.css" }
   let(:gemfile) { IO.read("#{project_path}/Gemfile") }
   let(:node_modules_file) { IO.read("#{project_path}/package.json") }
   let(:application_js_file) { IO.read("#{project_path}/app/javascript/application.js") }
   let(:layout_file) { IO.read("#{project_path}/app/views/layouts/application.html.erb") }
-  let(:application_css_file) { IO.read(application_css_path) }
+  let(:application_css_file) { "#{project_path}/app/javascript/css/application.css" ) }
```
Comment (monkeyci) [inline spec/features/front_end_spec.rb:13]: 🚫 **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Lint/Syntax: unexpected token tRPAREN

File: spec/features/front_end_spec.rb [hunk 3]
```diff
@@ -6,14 +6,14 @@
     remove_project_directory
   end
 
-  let(:application_css_path) { "#{project_path}/app/javascript/css/application.css" }
   let(:gemfile) { IO.read("#{project_path}/Gemfile") }
   let(:node_modules_file) { IO.read("#{project_path}/package.json") }
   let(:application_js_file) { IO.read("#{project_path}/app/javascript/application.js") }
   let(:layout_file) { IO.read("#{project_path}/app/views/layouts/application.html.erb") }
-  let(:application_css_file) { IO.read(application_css_path) }
+  let(:application_css_file) { "#{project_path}/app/javascript/css/application.css"}
```
Comment (monkeyci) [inline spec/features/front_end_spec.rb:13]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/SpaceInsideBlockBraces: Space missing inside }.

**Review by gmq — APPROVED** [review]

**Pull request merged**